pkg/grpc/device_definition.proto

```diff
@@ -151,6 +151,8 @@ message Integration {
     int32 ICE = 3;
     int32 PHEV = 4;
   }
+  optional int64 points = 9;
```

Comment:
Since we're requiring this in the DB, I have no qualms about "requiring" it here too.